packages/react-ui-core/src/IncrementDecrementFilter/IncrementDecrementFilter.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component, PropTypes } from 'react'
+
+export default class IncrementDecrementFilter extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      count: this.props.count,
+    }
+    this.increment = this.increment.bind(this)
+    this.decrement = this.decrement.bind(this)
+  }
+  increment() {
+    const incrementValue = this.state.count + 1
+    this.setState({
+      count: incrementValue,
+    })
+  }
+  decrement() {
+    if (this.state.count > 1) {
+      const decrementValue = this.state.count - 1
+      this.setState({
+        count: decrementValue,
+      })
+    }
+  }
+  render() {
+    const { count } = this.state
+    const { theme, label, leftUnit, rightUnit, id } = this.props
+    return (<div id={id} className={theme.container}>
+      <div className={theme.label}>{label}</div>
+      <div className={theme.componentContainer}>
+          <span onClick={this.increment} className={`leftUnit ${theme.contentInnards} ${theme.circle}`}>&#x002B;</span>
+          <span className={`${theme.contentInnards} ${theme.count}`}>{`${leftUnit} ${count} ${rightUnit}`}</span>
+          <span onClick={this.decrement} className={`rightUnit ${theme.contentInnards} ${theme.circle}`}>&#x002D;</span>
```

Comment:
The `+` and `-` children on the spans should probably be customizable.  Having them as defaults is fine, but the should allow for an arbitrary component / node.